ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt

```diff
@@ -129,19 +129,6 @@ class CloudFormationBrowserTest {
                     assertThat(createComplete).hasSize(1)
                     createComplete.first()
                 }
-
-                step("Check Logical ID action") {
-                    resource.click(MouseButton.RIGHT_BUTTON)
-                    findAndClick("//div[@text='Copy Logical ID']")
```

Comment:
the copy is actually in a few places I think ...

Reply:
We have it in cloudwatch logs as well, but I'm not sure why I put it in there. Copying multiple rows/columns works as expected